### DISCUSSION TITLE
Include option to access currently playing track properties by right-clicking on track title in playbar and album art

### DISCUSSION THREAD

**Discussion by takaji10** [TT-ReBORN/Georgia-ReBORN#21] [Q&A] [answered]
Title says it all on this one, it would be great to be able to right-click on the album art in the left panel or the track title of a currently-playing song in the playbar and access the properties dialogue window for that track.

The current right-click options in the album art viewer and the playbar are confusing right now - to me "Edit script," "Properties," and "Configure" all sound like the same thing. I would suggest keeping "Properties" reserved for the foobar2000 Properties dialogue window and changing "Configure" to "Panel Configuration."

**Comment by TT-ReBORN** [comment]
You can't change/do anything about it because it's the spider monkey panel ( component ) menu and the whole theme is one full spider monkey panel. If you want, you can disable it by enabling the option "Lock right click...".

**Comment by TT-ReBORN** [comment, marked as answer]
Hey @takaji10 ,

I've made a workaround about the SMP menu. I have created and added a new option menu for the album cover and 
disabled by default the SMP menu. I think this is the best solution and won't confuse new users anymore...

-TT
